appcues/src/main/java/com/appcues/trait/extensions/TargetRectangleInfoExt.kt

```diff
@@ -35,7 +41,27 @@ internal fun TargetRectangleInfo?.getRect(windowInfo: AppcuesWindowInfo): Rect?
     )
 }
 
-@Composable
 internal fun TargetRectangleInfo?.getContentDistance(): Dp {
     return this?.contentDistance?.dp ?: 0.0.dp
 }
+
+internal fun TargetRectangleInfo?.getTooltipPointerPosition(
```

Comment:
this piece of code determines the actual position of the tooltip, based on the preferred Tooltip position and the available space.

the result is TooltipPointerPosition